esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/install_env_file/bin/install_env_file.sh

```diff
@@ -3,15 +3,15 @@
 BASH_XTRACEFD=1
 set -eux
 
-# Copy the site specific environment file to the 'bin' directory in the
+# Copy the site specific environment file to the 'share/bin' directory in the
 # installed Cylc workflow (this directory is automatically added to the
 # ${PATH} by Cylc).
 SOURCE_PATH="${CYLC_WORKFLOW_RUN_DIR}/site/${SITE}-env"
-TARGET_DIR="${CYLC_WORKFLOW_RUN_DIR}/bin"
+TARGET_DIR="${CYLC_WORKFLOW_RUN_DIR}/share/bin"
```

Comment:
`${CYLC_WORKFLOW_SHARE_DIR}` can be used here instead of `${CYLC_WORKFLOW_RUN_DIR}/share/` 👍

Reply:
Fixed in commit [8d01587](https://github.com/ESMValGroup/ESMValTool/pull/3600/commits/8d01587e4901648286618161b6eafbb1d92a6fe4)